app/components/e-checkbox.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div :class="b(modifiers)">
+
+    <input
+      :aria-checked="checked ? 'true' : 'false'"
+      :class="b('field')"
+      :disabled="disabled"
+      :value="value"
+      :name="name"
+      :id="name"
+      v-bind="$attrs"
+      v-model="internalValue"
+      role="checkbox"
+      type="checkbox"
+      @blur="onBlur"
+      @change="onChange"
+      @focus="onFocus">
+
+    <label :for="name"
+           :class="b('label')"
+           @mouseenter="isHover = true"
+           @mouseleave="isHover = false">
```

Comment:
Do we need this `isHover` states here? Because we don't have a custom hover design

Reply:
Good question and to be honest I'm not 100% sure. I thought Patric, wanted to include it anyway in case we need it for further projects (re-usable).